src/views/Settings.vue

```diff
@@ -165,9 +157,6 @@ export default defineComponent({
     goToLaunchpad() {
       window.location.href = `${process.env.VUE_APP_LOGIN_URL}`
     },
-    getDateTime(time: any) {
-      return DateTime.fromMillis(time).toLocaleString(DateTime.DATETIME_MED);
```

Comment:
Remove the import for DateTime as it is not used.

Reply:
Done.